stablehlo/reference/Element.cpp

```diff
@@ -728,9 +736,9 @@ Element rem(const Element &e1, const Element &e2) {
         llvm::report_fatal_error("rem(bool, bool) is unsupported");
       },
       [](APFloat lhs, APFloat rhs) {
-        // APFloat::fmod VS APFloat:remainder: the returned value of the latter
-        // is not guaranteed to have the same sign as lhs. So mod() is preferred
-        // here. The returned "APFloat::opStatus" is ignored.
+        // APFloat::fmod VS APFloat:remainder: the returned value of the
```

Comment:
Was this change intended? The original comment is within the 80-char limit.

Reply:
Apologies, must have been the IDE auto correcting.